keyboards/yanghu/unicorne/keymaps/default/keymap.c

```diff
@@ -0,0 +1,99 @@
+                 KC_LCTL, KC_LALT, LOWER, KC_ENT,    KC_BSPC, KC_SPC, RAISE, KC_RGUI
+  ),
+
+  [_LOWER] = LAYOUT_unicorne(
+    _______,  KC_EXLM,  KC_AT,    KC_HASH,  KC_DLR,    KC_PERC,            KC_CIRC,  KC_AMPR,  KC_ASTR,  KC_LPRN,  KC_RPRN,  KC_DEL,
+    _______,  KC_UNDS,  KC_PLUS,  KC_LCBR,  KC_RCBR,   KC_PIPE,            KC_LEFT,  KC_DOWN,  KC_UP,    KC_RGHT,  KC_GRV,   KC_TILD,
+    _______,  KC_ESC,   KC_LGUI,  KC_LALT,  KC_CAPS,   KC_DQUO,            KC_HOME,  KC_END,   KC_PGUP,  KC_PGDN,  KC_PSCR,  RSFT_T(KC_SPC),
+             _______,  _______,   _______,   _______,  _______,   _______,   _______,  _______
+  ),
+
+  [_RAISE] = LAYOUT_unicorne(
+    _______,  KC_1,     KC_2,     KC_3,     KC_4,      KC_5,               KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_DEL,
+    _______,  KC_MINS,  KC_EQL,   KC_LBRC,  KC_RBRC,   KC_BSLS,            KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,
+    _______,  KC_ESC,   KC_RGUI,  KC_RALT,  KC_CAPS,   KC_QUOT,            KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,
+             _______,  _______,   _______,   _______,  _______,   _______,   _______,  _______
+  ),
+
+  [_ADJUST] = LAYOUT_unicorne(
+    RGB_VAI,   RGB_SAI, RGB_HUI,  RGB_MOD,  XXXXXXX,   RGB_TOG,            XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+    RGB_VAD,   RGB_SAD, RGB_HUD,  RGB_RMOD, XXXXXXX,   XXXXXXX,            XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+    XXXXXXX,   XXXXXXX, XXXXXXX,  XXXXXXX,  XXXXXXX,   XXXXXXX,            RESET,    XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+             _______,  _______,   _______,   _______,  _______,   _______,   _______,  _______
+  ),
+};
+// clang-format on
+layer_state_t layer_state_set_user(layer_state_t state) { return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST); }
+
+bool led_update_user(led_t led_state) {
+#ifdef OLED_DRIVER_ENABLE
+    oled_render_capslock(led_state.caps_lock);
+#endif
+    return true;
+}
+
+// Left encoder scrolls the mousewheel. Right encoder adjusts underglow hue.
+void encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) {
+        if (clockwise) {
+#ifdef MOUSEKEY_ENABLE
+            tap_code(KC_MS_WH_DOWN);
+#else
+            tap_code(KC_PGDN);
+#endif
+        } else {
+#ifdef MOUSEKEY_ENABLE
+            tap_code(KC_MS_WH_UP);
+#else
+            tap_code(KC_PGUP);
+#endif
+        }
+    } else {  // index = 1: right encoder
+        if (clockwise) {
+#ifdef RGB_MATRIX_ENABLE
+            rgb_matrix_step();
+#else
+            rgblight_increase_hue_noeeprom();
+#endif
+        } else {
+#ifdef RGB_MATRIX_ENABLE
+            rgb_matrix_step_reverse();
+#else
+            rgblight_decrease_hue_noeeprom();
+#endif
+        }
+    }
+}
+
+// Set underglow color to blue.
+void keyboard_post_init_user(void) { rgblight_sethsv(HSV_BLUE); }
```

Comment:
```suggestion
void keyboard_post_init_user(void) { rgblight_sethsv_noeeprom(HSV_BLUE); }
```

Also, you could use a define to set the default hue, iirc.

Reply:
fixed in c4ea096